src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const rows = table.rows;
+const head = rows[0];
+const people = [];
+const tBody = table.tBodies[0].children;
+
+for (const row of rows) {
+  const person = {};
+
+  person.name = row.cells[0].textContent;
+  person.position = row.cells[1].textContent;
+  person.age = row.cells[2].textContent;
+  person.salary = row.cells[3].textContent;
+  people.push(person);
+}
+people.pop();
+people.shift();
```

Comment:
The use of `people.pop()` and `people.shift()` removes the last and first elements of the `people` array, respectively. Ensure this behavior is intended, as it modifies the data set used for sorting and displaying.